node/service/src/service/dvm.rs

```diff
@@ -115,7 +115,8 @@ where
 					substrate_backend.clone(),
 					dvm_backend.clone(),
 					3,
-					0,
+					// TODO: improve this later
+					if network == "Crab" { 4969901 } else { 9100000 },
```

Comment:
Check when Darwinia will perform the runtime upgrade.

Reply:
Yeah, we need to tweak this value when the release date is confirmed.